lib/plugins/create/templates/tencent-nodejs8/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "tencent-nodejs",
+  "version": "1.0.0",
+  "description": "Tencent Serverless Cloud Function example using Nodejs",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "cloud.tencent.com",
+  "license": "MIT",
+  "devDependencies": {
+    "serverless-tencent-scf": "*"
```

Comment:
It'll be good to pin it to current version so `^0.1.7`. That will prevent any eventual breaking changes to a plugin to affect the template

Reply:
Thank you.
I have added the plugin version in package.json just now.